app/views/articles/_search.html.erb

```diff
@@ -59,6 +59,16 @@
     }
   }
 
+  function addOptionalBanner(query, displayBanner, jobsUrl) {
+    if (query) {
+      var lowerCaseQuery = query.toLowerCase();
+
+      if (displayBanner && (lowerCaseQuery == "job" || lowerCaseQuery == "jobs")) {
+        document.getElementById("articles-list").innerHTML == '<div class="crayons-notice crayons-notice--info mb-3 fs-base">Interested in joining our team? Explore our <a href=' + jobsUrl+ '>open roles</a>.</div>';
```

Comment:
This line throws me off a bit. Double equal is assigning the notice to the `innerHTML`? 👀

Reply:
oh typo! let me fix that, thanks!

Reply:
Settled for quirk instead of magic then! I was surprised too 😂